### DISCUSSION TITLE
Add new route by API

### DISCUSSION THREAD

**Discussion by techwavedev** [grafana/grafana#38840] [Alerting]
How can I add a new route by API without having to rewrite everything? So just making and append to the existing configuration or update the part  that changes regards to? 

from http://localhost:3000/api/alertmanager/grafana/config/api/v1/alerts

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
